### PR TITLE
IOS build error fix

### LINKED ISSUE #699: Remote wallet not working on iOS build
**Describe the bug**
After the build is a success in XCode and you try to run it in a real iPhone device or in your Mac, you can't login using a remote wallet.

**To Reproduce**
1. Switch platform to iOS
2. Unselect "SampleLogin - Web3Auth" scene in Build Settings
3. Click on Build and Run
4. Wait for build to finish in Unity
5. Switch your development profile in Xcode
6. If you have XCode 15.0.1 you need to add the flag "-ld_classic" in **Other Linker Flags**
7. Start the build in Xcode selecting a real device or simulator
8. Click on "Continue with Existing Wallet" button or select "Redirect to Wallet"

**Expected behavior**
QR Code is generated or is redirected to the selected wallet

**Mobile (please complete the following information):**
 - Chainsafe SDK version: [main]
 - Platform: [iOS]
 - Type: [Real device]
 - iOS version [17.0.3]

**Additional context**
```
[Web3] Core already initialized
Stacktrace is not supported on this platform.

[Web3] [Relayer] Checking for established connection
Stacktrace is not supported on this platform.

[Web3] [Relayer] Restarting transport
Stacktrace is not supported on this platform.

[Web3] [Relayer] Sending request through provider
Stacktrace is not supported on this platform.

[Web3] [JsonRpcProvider] Checking if connected
Stacktrace is not supported on this platform.
```
```
Web3Exception: HTTP.POST to https://api.gaming.chainsafe.io/logging/logEvent responded with error: <html>

<head><title>504 Gateway Time-out</title></head>

<body>

<center><h1>504 Gateway Time-out</h1></center>

</body>

</html>

 at ChainSafe.Gaming.Web3.Core.Unity.UnityHttpClient.UnityWebRequestToNetworkResponse (UnityEngine.Networking.UnityWebRequest request) [0x00000] in <00000000000000000000000000000000>:0 
 at ChainSafe.Gaming.Web3.Core.Unity.UnityHttpClient+<>c__DisplayClass4_0.<PostRaw>b__0 () [0x00000] in <00000000000000000000000000000000>:0 
 at System.Threading.ExecutionContext.RunInternal (System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, System.Object state, System.Boolean preserveSyncCtx) [0x00000] in <00000000000000000000000000000000>:0 
 at System.Runtime.CompilerServices.AsyncMethodBuilderCore+MoveNextRunner.Run () [0x00000] in <00000000000000000000000000000000>:0 
 at IEnumeratorAwaitExtensions+InstructionWrappers+<ReturnSelf>d__2`1[T].MoveNext () [0x00000] in <00000000000000000000000000000000>:0 
 at UnityEngine.SetupCoroutine.InvokeMoveNext (System.Collections.IEnumerator enumerator, System.IntPtr returnValueAddress) [0x00000] in <00000000000000000000000000000000>:0 
--- End of stack trace from previous location where exception was thrown ---

 at ChainSafe.Gaming.Evm.Unity.Dispatcher+<>c__DisplayClass6_0`1[T].<EnqueueTask>g__WrappedAction|0 () [0x00000] in <00000000000000000000000000000000>:0 
 at System.Threading.ExecutionContext.RunInternal (System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, System.Object state, System.Boolean preserveSyncCtx) [0x00000] in <00000000000000000000000000000000>:0 
 at System.Runtime.CompilerServices.AsyncMethodBuilderCore+MoveNextRunner.Run () [0x00000] in <00000000000000000000000000000000>:0 
 at System.Threading.Tasks.AwaitTaskContinuation.RunCallback (System.Threading.ContextCallback callback, System.Object state, System.Threading.Tasks.Task& currentTask) [0x00000] in <00000000000000000000000000000000>:0 
 at System.Threading.Tasks.Task.FinishContinuations () [0x00000] in <00000000000000000000000000000000>:0 
 at System.Threading.Tasks.Task.TrySetException (System.Object exceptionObject) [0x00000] in <00000000000000000000000000000000>:0 
 at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1[TResult].SetException (System.Exception exception) [0x00000] in <00000000000000000000000000000000>:0 
 at ChainSafe.Gaming.Web3.Core.Unity.UnityHttpClient+<>c__DisplayClass4_0.<PostRaw>b__0 () [0x00000] in <00000000000000000000000000000000>:0 
 at System.Threading.ExecutionContext.RunInternal (System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, System.Object state, System.Boolean preserveSyncCtx) [0x00000] in <00000000000000000000000000000000>:0 
 at System.Runtime.CompilerServices.AsyncMethodBuilderCore+MoveNextRunner.Run () [0x00000] in <00000000000000000000000000000000>:0 
 at IEnumeratorAwaitExtensions+InstructionWrappers+<ReturnSelf>d__2`1[T].MoveNext () [0x00000] in <00000000000000000000000000000000>:0 
 at UnityEngine.SetupCoroutine.InvokeMoveNext (System.Collections.IEnumerator enumerator, System.IntPtr returnValueAddress) [0x00000] in <00000000000000000000000000000000>:0 
--- End of stack trace from previous location where exception was thrown ---

 at ChainSafe.Gaming.Web3.Analytics.ApiAnalyticsClient.CaptureEvent (ChainSafe.Gaming.Web3.Analytics.AnalyticsEvent eventData) [0x00000] in <00000000000000000000000000000000>:0 
 at System.Threading.ExecutionContext.RunInternal (System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, System.Object state, System.Boolean preserveSyncCtx) [0x00000] in <00000000000000000000000000000000>:0 
 at System.Runtime.CompilerServices.AsyncMethodBuilderCore+MoveNextRunner.Run () [0x00000] in <00000000000000000000000000000000>:0 
 at System.Threading.Tasks.AwaitTaskContinuation.RunCallback (System.Threading.ContextCallback callback, System.Object state, System.Threading.Tasks.Task& currentTask) [0x00000] in <00000000000000000000000000000000>:0 
 at System.Threading.Tasks.Task.FinishContinuations () [0x00000] in <00000000000000000000000000000000>:0 
 at System.Threading.Tasks.Task.TrySetException (System.Object exceptionObject) [0x00000] in <00000000000000000000000000000000>:0 
 at System.Threading.Tasks.TaskCompletionSource`1[TResult].TrySetException (System.Exception exception) [0x00000] in <00000000000000000000000000000000>:0 
 at ChainSafe.Gaming.Evm.Unity.Dispatcher+<>c__DisplayClass6_0`1[T].<EnqueueTask>g__WrappedAction|0 () [0x00000] in <00000000000000000000000000000000>:0 
 at System.Threading.ExecutionContext.RunInternal (System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, System.Object state, System.Boolean preserveSyncCtx) [0x00000] in <00000000000000000000000000000000>:0 
 at System.Runtime.CompilerServices.AsyncMethodBuilderCore+MoveNextRunner.Run () [0x00000] in <00000000000000000000000000000000>:0 
 at System.Threading.Tasks.AwaitTaskContinuation.RunCallback (System.Threading.ContextCallback callback, System.Object state, System.Threading.Tasks.Task& currentTask) [0x00000] in <00000000000000000000000000000000>:0 
 at System.Threading.Tasks.Task.FinishContinuations () [0x00000] in <00000000000000000000000000000000>:0 
 at System.Threading.Tasks.Task.TrySetException (System.Object exceptionObject) [0x00000] in <00000000000000000000000000000000>:0 
 at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1[TResult].SetException (System.Exception exception) [0x00000] in <00000000000000000000000000000000>:0 
 at ChainSafe.Gaming.Web3.Core.Unity.UnityHttpClient+<>c__DisplayClass4_0.<PostRaw>b__0 () [0x00000] in <00000000000000000000000000000000>:0 
 at System.Threading.ExecutionContext.RunInternal (System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, System.Object state, System.Boolean preserveSyncCtx) [0x00000] in <00000000000000000000000000000000>:0 
 at System.Runtime.CompilerServices.AsyncMethodBuilderCore+MoveNextRunner.Run () [0x00000] in <00000000000000000000000000000000>:0 
 at IEnumeratorAwaitExtensions+InstructionWrappers+<ReturnSelf>d__2`1[T].MoveNext () [0x00000] in <00000000000000000000000000000000>:0 
 at UnityEngine.SetupCoroutine.InvokeMoveNext (System.Collections.IEnumerator enumerator, System.IntPtr returnValueAddress) [0x00000] in <00000000000000000000000000000000>:0 
--- End of stack trace from previous location where exception was thrown ---

 at System.Runtime.CompilerServices.AsyncMethodBuilderCore+<>c.<ThrowAsync>b__7_0 (System.Object state) [0x00000] in <00000000000000000000000000000000>:0 
 at UnityEngine.UnitySynchronizationContext+WorkRequest.Invoke () [0x00000] in <00000000000000000000000000000000>:0 
 at UnityEngine.UnitySynchronizationContext.Exec () [0x00000] in <00000000000000000000000000000000>:0
```

### PR DESCRIPTION
**Pull request by robGG1997**
Closes #699 

Make sure Strip Engine Code is set to false and Code Stripping is Set to minimal in Player Settings

### CODE REVIEW

**Review by sneakzttv — APPROVED** [review]

**Review by juans-chainsafe — APPROVED** [review]
Issue is fixed! But is not working the “Redirect to Wallet” it redirects to the wallet but it doesnt prompt to connect to the app

File: Packages/io.chainsafe.web3-unity/Tests/Runtime/EvmTests.cs [hunk 1]
```diff
@@ -333,7 +333,7 @@ public IEnumerator TestTransferErc721()
     [UnityTest]
     public IEnumerator TestTransferErc1155()
     {
-        config.TestResponse = "0x5de8fb6c522d4ba85f09961f03bbdc1ee8d9c283ddd1dea0f3ccdfa27cc189af";
+        config.TestResponse = "0x390b47d378e9a6de830e2cc6d624de0920efc44d7b40fb61f75d983545c987fc";
```
Comment (kalambet) [inline Packages/io.chainsafe.web3-unity/Tests/Runtime/EvmTests.cs:336]: Is this result of the Sepolia change?

Comment (sneakzttv) [reply Packages/io.chainsafe.web3-unity/Tests/Runtime/EvmTests.cs:336]: yes

Comment (robGG1997) [reply Packages/io.chainsafe.web3-unity/Tests/Runtime/EvmTests.cs:336]: Yeah, the test was failing so I replaced the hash

**Review by kalambet — APPROVED** [review]

**Pull request merged**